<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculo COM Torque</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>9</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="liveView">
        <property name="cursor">
         <cursorShape>BlankCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="l_detail">
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchi">
    <property name="title">
     <string>Inicio</string>
    </property>
    <addaction name="actionCargar"/>
    <addaction name="actionExportar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="controles">
   <property name="minimumSize">
    <size>
     <width>336</width>
     <height>614</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>397</width>
     <height>654</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Controles</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Posición:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_posiciones"/>
       </item>
       <item>
        <widget class="QPushButton" name="b_anadirPos">
         <property name="text">
          <string>Añadir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="segmentos">
        <attribute name="title">
         <string>Segmentos</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QPushButton" name="b_muslo">
           <property name="text">
            <string>Muslo</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="b_antebrazo">
           <property name="text">
            <string>Antebrazo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="b_cabeza">
           <property name="text">
            <string>Cabeza</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="b_tronco">
           <property name="text">
            <string>Tronco</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="b_brazo">
           <property name="text">
            <string>Brazo</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="b_mano">
           <property name="text">
            <string>Mano</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="b_pantorrilla">
           <property name="text">
            <string>Pantorrilla</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="b_pie">
           <property name="text">
            <string>Pie</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="dimensiones">
        <attribute name="title">
         <string>Dimesiones</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Sexo:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cbTablaDimensiones">
             <item>
              <property name="text">
               <string>Hombre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mujer</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tablaDimensiones">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DotLine</enum>
           </property>
           <row>
            <property name="text">
             <string>Cabeza</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Tronco</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Brazo</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Antebrazo</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Mano</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Muslo</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Pantorrilla</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Pie</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>COM</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Masa</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="b_torques">
         <property name="text">
          <string>Calcular Torques</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
    <zorder>cb_posiciones</zorder>
    <zorder>label_5</zorder>
    <zorder>b_anadirPos</zorder>
    <zorder>b_cabeza</zorder>
    <zorder>b_brazo</zorder>
    <zorder>b_antebrazo</zorder>
    <zorder>tabWidget</zorder>
    <zorder>horizontalSpacer_6</zorder>
    <zorder>b_torques</zorder>
   </widget>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="action_simple_gui">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>GUI simplificada</string>
   </property>
   <property name="toolTip">
    <string>Menos controles</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
